docs/userguides/config.md

```diff
@@ -384,14 +384,22 @@ To learn more about how request headers work in Ape, see [this section of the Ne
 
 ## Testing
 
-Configure your test accounts:
+Configure your test accounts and test folder:
 
 ```yaml
 test:
   mnemonic: test test test test test test test test test test test junk
   number_of_accounts: 5
+
+# Optional: Configure the tests folder location
+tests_folder: tests
```

Comment:
I feel like explaining the test folders should come after the test accounts, as a separate topic.